pkgs/network/src/interface.zig

```diff
@@ -149,8 +149,8 @@ pub const GossipTopic = enum {
 };
 
 pub const GossipMessage = union(GossipTopic) {
-    block: types.SignedBeamBlock,
-    vote: types.SignedVote,
+    block: types.SignedBlockWithAttestation,
+    vote: types.SignedAttestation,
```

Comment:
```suggestion
    attestation: types.SignedAttestation,
```

Also check if we have changed the gossip topic in specs from vote to attestation,

Reply:
It hasn't been updated in specs, I'll update it there later. Pushing the change here.